docker/Dockerfile.base

```diff
@@ -1,5 +1,7 @@
 FROM ubuntu:18.04
 
+ENV LANG C.UTF-8
```

Comment:
I've always seen this set to things that look like `en_US.utf-8` does the capitalization matter?